docs/get-support.md

```diff
@@ -2,21 +2,21 @@
 title: Get Support
 ---
 
-## Get Event Bus Support
+# Get Support
 
-The best way to request support from the Event Bus Team is to reach out to us via the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) Slack channel in the Digital Service VA workspace.
+The best way to request support from the Event Bus Team is to reach out to us via the [#ves-event-bus](https://dsva.slack.com/archives/C042ZQ7JUAX) Slack channel in the `Office of CTO @VA` workspace.
 
```

Comment:
Why are we putting Office of CTO @VA in special formatting?

Reply:
It's referencing a specific Slack name that may not be obvious. I also considered italicizing it, but I can remove the formatting. Marking as resolved.